src/tutorials/0006-anatomy-of-a-cid/06.md

```diff
@@ -22,7 +22,7 @@ Looking at the results from the CID Inspector tool we can see several parts that
 - `Multicodec`: `code` is the identifier of the codec, in this case `0x70` for `dag-pb`, an IPLD format
```

Comment:
This changed in the screenshot from 0x70 to 0x112 in the screenshot, so needs to be updated here to match. Do we know why it changed? We started with the same CID, right?

Reply:
This is bug in the CID tool currently being fixed: https://github.com/multiformats/cid-utils-website/pull/36
I'll update the screenshot by tampering with the html code to have the correct values.
I'll also try to tackle the PR fix.

Reply:
Screenshot fixed.

Reply:
Bug is fixed on the website as well 🎉